runtime/pkg/conncache/conncache.go

```diff
@@ -62,6 +62,8 @@ type Options struct {
 	HangingFunc func(cfg any, open bool)
 	// Metrics are optional instruments for observability.
 	Metrics Metrics
+	// The maximum time the error stays in cache
+	ErrorTimeout time.Duration
```

Comment:
1. It looks like this is not set anywhere?
2. Consider calling it `ErrorTTL` instead since it's not a timeout the same way that `OpenTimeout` and `CloseTimeout` are

Reply:
Done.